topics/installation/README.md

```diff
@@ -2,66 +2,17 @@
 
 ## Packages
 
-Pre-built packages are available for CentOS 6/7/8, Ubuntu 14.04/16.04/18.04 and
-Windows 7 SP1 and above.
+Pre-built packages are available for CentOS 7, Ubuntu 20.04/22.04 and
+Rocky Linux 8 and 9.  All packages are available from our
```

Comment:
@absurdfarce Did we miss mentioning Windows packages here?

Reply:
Yes, will add.